src/tools/dart-compile.md

```diff
@@ -57,6 +57,10 @@ The `dart compile` command replaces the
   see the [platforms discussion](/overview#platform).
 {{site.alert.end}}
 
+Refer to the [native_app] sample for a simple example of using `dart create`
```

Comment:
I must be context switching too much, thanks for catching this...

Reply:
> I must be context switching too much, thanks for catching this...

I get it! Since I'd just reviewed the `dart create` page), it took a minute for me to realize that `dart create` wasn't right. :D